ex03.ex

```diff
@@ -55,7 +55,11 @@ defmodule Ex03 do
 
   """
 
-  def odd_even . . . "your code"
+  def odd_even([h | t]) when Integer.is_odd(h), do: [:odd | odd_even(t)]
```

Comment:
## -2

The last function head should be `def odd_even([]), do: []`.

As it stands, it works, but it also allows you to write  `odd_even(:wombat)`